src/app/public/modules/i18n/resources.service.ts

```diff
@@ -67,12 +69,25 @@ export class SkyAppResourcesService {
   /**
    * Gets a resource string based on its name.
    * @param name The name of the resource string.
+   * @param args Any templated args.
    */
   public getString(name: string, ...args: any[]): Observable<string> {
-    if (!this.resourcesObs) {
-      const localeObs = this.localeProvider.getLocaleInfo();
+    const localeObs: Observable<SkyAppLocaleInfo> = this.localeProvider.getLocaleInfo();
+    return this.getStringForLocaleInfoObservable(localeObs, name, ...args);
+  }
+
+  /**
+   * Gets a resource string for a specific locale based on its name.
+   * @param localeInfo The locale to use.
+   * @param name The name of the resource string.
+   * @param args Any templated args.
+   */
+  public getStringForLocale(localeInfo: SkyAppLocaleInfo, name: string, ...args: any[]): Observable<string> {
```

Comment:
Let's swap the order of the `localeInfo` and `name` parameters so `name` remains the first parameter.

Reply:
The reason I went with that order is because `SkyLibResourcesService` already had a `getStringForLocale()` with the arguments in this order:

https://github.com/blackbaud/skyux-i18n/blob/251fa6244737350b60e6b1083b32060ef485f3b6/src/app/public/modules/i18n/lib-resources.service.ts#L59

Would you still like me to swap the order for this method or should we leave it so it's consistent with `SkyLibResourcesService.getStringForLocale()`?

Reply:
Good catch. Let's leave the order you have so it's consistent with this method.